_data/toc.yaml

```diff
@@ -1339,9 +1339,6 @@ manuals:
       title: Install binaries
   - path: /engine/install/linux-postinstall/
     title: Optional post-installation steps
-  - path: /engine/release-notes/
-    title: Release notes
-  - sectiontitle: Previous versions
     section:
     - path: /engine/release-notes/19.03/
       title: Engine 19.03 release notes
```

Comment:
You need to move the section itself as well (looks like currently only the section title was moved)

Reply:
Ohh.... interesting; so because of this, the TOC shows "undefined", and now I wonder if something like this is related to https://github.com/docker/docker.github.io/issues/13553

<img width="559" alt="Screenshot 2022-08-11 at 14 23 36" src="https://user-images.githubusercontent.com/1804568/184132794-f82cebe5-22f5-4c37-bb8a-60c9400fe048.png">

Reply:
Now we know what happens :). So the section was there but the indentation was off.